setup.cfg

```diff
@@ -36,7 +36,13 @@ install_requires = six
 setup_requires = setuptools>=44; setuptools_scm[toml]>=3.4.3
 
 [options.extras_require]
-test = astroid<=2.5.3; pytest
+test = 
+    astroid<=2.5.3
+    pytest
+    typing; python_version < "3.5"
```

Comment:
Isn't this an essential (i.e. not extra) requirement for Python 2 users?

Reply:
Yup! Fixed